test-packages/e2e-tests/start.js

```diff
@@ -0,0 +1,8 @@
+var process = require('child_process');
+process.exec('cds run',function (err,stdout,stderr) {
```

Comment:
It looks like there is no linter running on this ;) (We don't need to do this right away)